src/Couchbase.Lite.Shared/API/Database/Collection.cs

```diff
@@ -278,12 +321,26 @@ public bool Save(MutableDocument document, Func<MutableDocument, Document, bool>
         /// doesn't exist</exception>
         public DateTimeOffset? GetDocumentExpiration(string docId)
         {
-            throw new NotImplementedException();
+            if (LiteCoreBridge.Check(err => Native.c4coll_getDoc(c4coll, docId, true, C4DocContentLevel.DocGetCurrentRev, err)) == null) {
```

Comment:
Memory leak, return value of c4coll_getDoc never released.

Reply:
released c4doc if it's found in collection.